server/spec/services/rpc/node_service_pod_handler_spec.rb

```diff
@@ -10,15 +10,14 @@
       allow(subject).to receive(:migration_done?).and_return(true)
     end
 
-    it 'returns hash with error if id does not exist' do
-      list = subject.list('foo')
-      expect(list[:error]).not_to be_nil
+    it 'fails if node does not exist' do
+      expect{subject.list('foo')}.to raise_error RuntimeError, 'Node not found'
     end
 
-    it 'returns hash with error if migration is not done' do
-      allow(subject).to receive(:migration_done?).and_return(false)
```

Comment:
This spec was broken, it was failing on the `node_id: foo` and never checking `migration_done?`.